backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -165,7 +167,7 @@ public static List<FeedbackMessage> validateState(ValueOrError input) {
   }
 
   public static List<FeedbackMessage> validateCountry(ValueOrError input) {
-    return validateInSet(input, COUNTRY_CODES);
+    return validateInSet(input, VALID_COUNTRY_CODES);
```

Comment:
Might be a silly question but why not instead of converting the list to lower case we just convert the input in line 170 to upper?

Reply:
The validateInSet method converts the input value back to lowercase (line 325). That's needed for cases when people put something like "Female" instead of "FEMALE" - the set values for those are stored as lowercase.